addons/jenkins/1.x/jenkins.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: jenkins
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.9.4-1"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.9.4-2"
```

Comment:
I'm wondering if this should not be `2.5.4-1` with such a chart version bump 🤔.

Reply:
Yeah, it is like that in the master branch. Doesn't seem right though.